### ISSUE TITLE
Halt: a rainbow screen of death

### ISSUE THREAD

**Issue by audetto** [AppleWin/AppleWin#961]
I was messing around with A2 GameServer and ended up constantly in a `hlt` instruction.

I seem to understand that there is no way out.
Would it make any sense to trap it and inform the user that the CPU is dead?

**Comment by tomcw** [comment]
>Would it make any sense to trap it and inform the user that the CPU is dead?

Yes... but I feel this is fairly nuanced (rare) since it's 6502-only (not 65C02), so you'd have to be running with a non-default Apple II model (ie. not the Enhanced //e).

I'd imagine it'd be fairly simple to display a MessageBox for this instruction.
